static/lang/en.json

```diff
@@ -13,24 +13,35 @@
   "ATTRIBUTE.INSIGHT": "Insight",
   "ATTRIBUTE.MIGHT": "Might",
   "ATTRIBUTE.WILLPOWER": "Willpower",
+  "BIO.ABSORBS": "AB: ",
+  "BIO.AFFINITIES": "Affinities",
+  "BIO.ATTACKS": "Attacks",
   "BIO.ATTRIBUTES_AND_STATUS_EFFECTS": "Attributes and Status Effects",
   "BIO.DEFENSE": "Defense:",
   "BIO.EXPERIENCE": "Experience",
   "BIO.EXPERIENCE_AND_LEVEL": "Experience & Level",
   "BIO.FABULA_POINTS": "Fabula Points ",
   "BIO.HIT_POINTS": "Hit Points:",
   "BIO.IDENTITY": "Identity:",
+  "BIO.IMMUNE": "IM: ",
+  "BIO.INIT": "Init:",
```

Comment:
Just calling this out since I just added `INITIATIVE_MODIFIER` while this was being worked on. Do we want to settle on one of the two, or does it make sense to have them be different in different places?

Reply:
It's definitely okay to have both - I chose mine based on the monster sheet in the book